.ci/coverage.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+# this is in a separate script from `.travis.yml` so we can set this option.
+shopt -s extglob
```

Comment:
always `set -eu` unless it's really clear that this isn't necessary.

Reply:
this script is actually no longer used – I should rm it